### PR TITLE
tests: use full path to test-snapd-refresh.version binary

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Add full path to the test-snapd-refresh.version binary

This is the error running with external backend:

tests.session -u test exec test-snapd-refresh.version
/tmp/tests.session-4f317b13-6145-488d-9d94-b962b8dc6017/exec: 2: exec:
test-snapd-refresh.version: not found
grep error: pattern not found, got:

### CODE REVIEW

**Review by zyga — CHANGES_REQUESTED** [review]

**Comment by zyga** [comment]
I think this shows another problem though: we lack the right path entry somehow. CC @bboozzoo for ideas.

File: tests/main/refresh-app-awareness/task.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ execute: |
 
     # We can run "snap run" as a test user, which does not have the permission
     # to create the run inhibition lock file.
-    tests.session -u test exec test-snapd-refresh.version | MATCH v1
+    tests.session -u test exec /snap/bin/test-snapd-refresh.version | MATCH v1
```
[the remark this answers is not in the document]
Comment (sergiocazzolato) [reply tests/main/refresh-app-awareness/task.yaml:50]: @zyga you mean to do:
su -c 'snap run test-snapd-refresh.version' test | MATCH v1 ?

[the remark this answers is not in the document]
Comment (zyga) [reply tests/main/refresh-app-awareness/task.yaml]: No, I mean `tests.session` that uses `snap run test-snapd-refresh.version`

[the remark this answers is not in the document]
Comment (sergiocazzolato) [reply tests/main/refresh-app-awareness/task.yaml]: @zyga, updated, thanks for the review

**Review by anonymouse64 — APPROVED** [review]
LGTM, with the caveat that we add a TODO to investigate why this is necessary somewhere in tests.session

**Review by zyga — APPROVED** [review]
LGTM, I'll investigate the PATH issue tomorrow.

**Pull request merged**